server/neptune/workflows/internal/pr/runner.go

```diff
@@ -61,6 +69,9 @@ func newRunner(ctx workflow.Context, scope workflowMetrics.Scope, tfWorkflow rev
 		ShutdownSignalChannel: workflow.GetSignalChannel(ctx, ShutdownSignalID),
 		Scope:                 scope,
 		RevisionProcessor:     &revisionProcessor,
+		// TODO: make these configurations
+		InactivityTimeout: time.Hour * 24 * 7,
+		ShutdownPollTick:  time.Hour * 24,
 
 		cancel: func() {},
```

Comment:
Why is this a global? let's not use globals if we don't have to. Since onNewRevision is only called once, you can probably just call cancel, newRevisionCtx := workflow.WithCancel(ctx) before passing it in, and then locally store the cancel function.  I know i told you to not have multiple ctx being tracked in one function, but in this case i think it's better than a global var.

Reply:
Sure I'll move back to giving the ctx a name